src/antsibull/build_changelog.py

```diff
@@ -551,16 +551,22 @@ def _append_base_porting_guide_bytes(builder: RstBuilder, changelog: Changelog)
     def _get_porting_guide_bytes(changelog: Changelog) -> bytes:
         if changelog.ansible_version.major > 2:
             version = f"{changelog.ansible_version.major}"
-            base_version = changelog.base_collector.latest
-            base_version = f"{base_version.major}.{base_version.minor}"
+            core_version = changelog.core_collector.latest
+            core_version = f"{core_version.major}.{core_version.minor}"
         else:
             version = f"{changelog.ansible_version.major}.{changelog.ansible_version.minor}"
-            base_version = f"{changelog.ansible_version.major}.{changelog.ansible_version.minor}"
+            core_version = f"{changelog.ansible_version.major}.{changelog.ansible_version.minor}"
         builder = RstBuilder()
-        builder.add_raw_rst(
-            f"..\n"
-            f"   THIS DOCUMENT IS AUTOMATICALLY GENERATED BY ANTSIBULL! PLEASE DO NOT EDIT"
-            f" MANUALLY! (YOU PROBABLY WANT TO EDIT porting_guide_base_{base_version}.rst)\n")
+        if changelog.ansible_version.major < 4:
+            builder.add_raw_rst(
+                f"..\n"
+                f"   THIS DOCUMENT IS AUTOMATICALLY GENERATED BY ANTSIBULL! PLEASE DO NOT EDIT"
+                f" MANUALLY! (YOU PROBABLY WANT TO EDIT porting_guide_base_{core_version}.rst)\n")
```

Comment:
Just double-checking that `base_{core_version}` is correct here. I think it is, but just in case . . .

Reply:
It's correct, since this is only used if `changelog.ansible_version.major < 4`: Ansible 2.10 and Ansible 3 are based on ansible-base 2.10.